### ISSUE TITLE
GET request to kongruent.xyz blocked by AdBlock

### ISSUE THREAD

**Issue by tohaklim** [ENT8R/NotesReview#71]
Not really an issue for me, but it might be something to flag with ad list maintainers.
I don't have ublock on this machine, so can't see detailed blocking logs, but here is the Edge console:
```
index.6660fcd7.js:1 GET https://notesreview.kongruent.xyz/api/search?status=open&sort_by=created_at net::ERR_BLOCKED_BY_CLIENT
v @ index.6660fcd7.js:1
search @ index.6660fcd7.js:1
pe @ index.6660fcd7.js:1
(anonymous) @ index.6660fcd7.js:1
async function (async)
(anonymous) @ index.6660fcd7.js:1
(anonymous) @ index.6660fcd7.js:1
```

Thanks!

**Comment by ENT8R** [comment]
Weird... I can't reproduce this. You said you can reproduce this when visiting the site using Edge. Which ad blocker addon do you have installed?

**Comment by ENT8R** [comment]
I think I found the cause behind it: https://github.com/easylist/ruadlist/blob/bad81acd021be625235fb2b1435d803c192ddbb6/advblock/first_level.txt#L16

Which says at the top of the list:
>  Бесплатные DNS-сервисы и не бесплатные домены первого уровня,
крайне часто использующихся для мошеннических схем,
распространения навязчивой рекламы и вирусов.

Automatic translation:
> Free DNS services and not free first level domains, very often used for fraudulent schemes, intrusive advertising and viruses.

As domains with `*.herokuapp.com` are excluded from this rule, I think it would also make sense to add `*.github.io` to the list of exlusions. I will see what I can do about that.

**Comment by ENT8R** [comment]
I opened a new issue in the repository which manages the filter list and it was already fixed just a few minutes after I reported it: https://github.com/easylist/ruadlist/issues/20
It likely takes some time for the list to be generated and downloaded by the user of the adblock addon, but generally I expect it to work now...
